### PR TITLE
Improve install guide

### PR DESCRIPTION
**Pull request by themightychris**
Still needs some work but I found install.txt a bit hard to follow and lacking in setting up on a fresh Debian 7 box, this is my work-in-progress update to it

### CODE REVIEW

**Comment by clausfuglsang** [comment]
Hey Chris.
You don't mention anything about where the ebot-web folder should be moved to. Shouldn't it be moved to /var/www or what do you do with it?

**Comment by KoSoVaR** [comment]
It can be wherever you want it to be as long as the web server directive is
pointing to that directive.  A lot of this just assumes you have some
exposure to configuring a LAMP stack

**Comment by clausfuglsang** [comment]
I have limited experience with configuration of LAMP or just apache2, so I'm forced to just follow a guide.
I've managed to get it pointing to the eBot folder, by following deStrO's guideline, so I can browse the first page of eBote. But admin.php or any other links of the site isn't functional yet.

**Comment by deStrO** [comment]
A new guide is available on the forum

**Comment by themightychris** [comment]
@deStrO installation instructions are best versioned with the code they're for, please consider linking to versioned install guides from the forum instead
